src/views/PopupView.tsx

```diff
@@ -37,6 +40,9 @@ export default function PopupView(
                     <dt>Controls Help</dt>
                     <dd onMouseOver={()=>setControlsHelpImgNum(0)}>camera</dd>
                     <dd onMouseOver={()=>setControlsHelpImgNum(1)}>selection</dd>
+                    <li><div>Body Segments
```

Comment:
`li` elements aren't supposed to be used outside of `ol`, `ul`, or `menu`

Reply:
fixed